bundles/CoreBundle/config/pimcore/default.yaml

```
@@ -190,6 +190,9 @@ pimcore:
                         textarea: \Pimcore\Model\Asset\MetaData\ClassDefinition\Data\Textarea
         type_definitions:
             map:
+                unknown:
+                    class: \Pimcore\Model\Asset\Unknown
+                    matching: [ "/\\.dwg/", "/\\.stp$/" ]
```

Should there be a `$` at the end?

```suggestion
                    matching: [ "/\\.dwg$/", "/\\.stp$/" ]
```

---

@jdreesen Good catch :)